### PR TITLE
thing

### PR DESCRIPTION
**Pull request by KMatias123**
the mods class is made similarly to what kami has, this is because i have always worked with kami based clients

### CODE REVIEW

**Comment by bebeli555** [comment]
The Mods class and the annotation registerMod thing instead of the constructor aren't really necessary as it works fine already but the other fixes and stuff were good.

**Comment by KMatias123** [comment]
i did it wrong lmao

**Comment by KMatias123** [comment]
i cant be bothered lol
